sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -322,6 +322,37 @@ def test_get_messages(self, resource_group, location, storage_account, storage_a
         self.assertIsInstance(message.expires_on, datetime)
         self.assertIsInstance(message.next_visible_on, datetime)
 
+    @GlobalStorageAccountPreparer()
+    def test_receive_one_message(self, resource_group, location, storage_account, storage_account_key):
```

Comment:
we should add a test case with empty queue.

Reply:
Done!